docs/readme-cmake.md

```diff
@@ -1,88 +1,56 @@
-First you need a copy of the code (if you intend on working on the game use your fork's URL here):
+# Build instructions
+
+First you need a copy of the code (if you intend on working on the game, use your fork's URL here):
 
 ```powershell
 > git clone https://github.com/endless-sky/endless-sky
 ```
 
 The game's root directory, where your `git clone`d files reside, will be your starting point for compiling the game.
 
-Next you will need to install a couple of dependencies to build the game.
-
-## Build Environment
-
-There are several different ways to build Endless Sky, depending on your operating system and preference.
-- [Windows](#Windows)
-- [MacOS](#MacOS)
-- [Linux](#Linux)
-
-## Windows (Visual Studio) <Windows>
-We are currently switching to using VS with cmake. If you wish the older MingW build instructions, they are located at the end of the Windows section.
-
-Download Visual Studio, and make sure to install the following features: 
-- "Desktop Development with C++", 
-- "C++ Clang Compiler for Windows", 
-- "C++ clang-cl for vXXX build tools", and 
-- "C++ CMake tools for Windows".
-
-Please note that it is recommened to use VS 2022 (or higher) for its better CMake integration.
-
-### Windows Build Process
-
-1. Open the endless-sky folder that you cloned in the initial step.
-2. Wait while Visual Studio loads everything. This may take a few minutes the first time, but should be relatively fast on subsequent loads.
-3. On your toolbar there should be a pulldown menu that says "debug" or "release." Select the version you want to build.
-4. Hit the "build" button, or find the "Build" menu and select "Build All"
-5. In the status window it will give a scrolling list of actions being completed. Wait until it states "Build Complete"
-6. At this point you should be able to launch the recently completed build by hitting the F5 key or the run button. The recently build executable and essential libraries can be found in your Endless-Sky folder, nested within a subfolder created by Visual Studio during the build process
-
-#### Notes
+Next, you will need to install a couple of dependencies to build the game. There are several different ways to build Endless Sky, depending on your operating system and preference. As an IDE, we recommend using [Visual Studio Code](#visual-studio-code).
```

Comment:
Maybe skip the mention of an IDE at this point?
When reading this, it looks like you're suggesting I follow that link, which takes me to the VSCode section near the end which talks about installing extensions. But, I never installed those dependencies. I don't think it'd be difficult for a new user to end up skipping the installation of the build tools and being very confused when VSCode can't build anything.
Instead of removing it entirely, you could also say something like: "Once you have these dependencies, you can move on to [building the game](link to the building section)."